src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -56,7 +66,11 @@ public JsonAdaptedPerson(Person source) {
         name = source.getName().fullName;
         phone = source.getPhone().value;
         email = source.getEmail().value;
+
+        parentPhone = source.getParentPhone().value;
+        parentEmail = source.getParentEmail().value;
         address = source.getAddress().value;
+        outstandingFee = source.getFee().value;
         remark = source.getRemark().value;
```

Comment:
maybe we can change this to fulfil SLAP also, like what you mentioned in tele😂🤔

Reply:
Its bad abstraction but OK SLAP. Regardless, there is a lot of refactoring to do, and i think we should do it in a separate PR after merging urs and my feature branch, to minimise merge conflicts